### DISCUSSION TITLE
Dev.json config file has been ignored in 10.9.0 version

### DISCUSSION THREAD

**Discussion by milica7** [cypress-io/cypress#24310] [Questions and Help]
Hi, 

we are having issues with multiple config files, we followed the instructions from https://docs.cypress.io/api/plugins/configuration-api#Switch-between-multiple-configuration-files, but when we run tests with something like this _cypress run --env configFile=dev, the config from the main file is loaded (cypress.config.js) and we can't see any error before that, so seems like the dev.json is not loaded at all.

**Comment by filiphric** [comment]
With Cypress v10, you are no longer able to load json files to your configuration. Basically, you have two options:

1. you can create a `dev.config.js` and load it like this `npx cypress open --config-file=dev.config.js` which will remove the configuration entirely
2. you can pass your own env flag to load the file you need. so e.g. use a command like this: `npx cypress open --env version=dev`
then, in your `cypress.config.js` you will have something like this:
```js
const { defineConfig } = require("cypress");

module.exports = defineConfig({
  e2e: {
    setupNodeEvents(on: Cypress.PluginEvents, config: Cypress.PluginConfigOptions) {
       // see if we passed --env version=dev
       if (config.env.version = 'dev') {
        // if we did, use dev.json file an put its content into env object, so we can access it via Cypress.env() function
         config.env = require('./dev.json')
      }
    }
  }
});

```
